addons/hr_attendance/views/hr_attendance_view.xml

```diff
@@ -173,6 +173,7 @@
                 <field name="employee_id" type="row"/>
                 <field name="check_in" interval="week" type="col"/>
                 <field name="worked_hours" type="measure" widget="float_time"/>
+                <field name="overtime_hours" type="measure" widget="float_time"/>
```

Comment:
by doing that you also change the initial active measure (see comment
// the last field with type="measure" (if any) will be used as measure else __count
in graph arch parser
)
So you should put your newly defined field a bit above

Reply:
(same remark everywhere)

Reply:
note that GROUPABLE_TYPES does not contain float, so it might be better to drop the attribute measure if you only need to set the attribute widget